cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "chromeWebSecurity": false
```

Comment:
This is needed for redirections from login SSO in HTTPS to our local s.io (not in HTTPS)